### ISSUE TITLE
Documentation about design system and accessibility is missing

### ISSUE THREAD

**Issue by bmlancien** [OpenEnergyPlatform/oeplatform#1709]
## Description of the issue

Documentation about design system and accessibility needs to be added in order to create more consistent design and improve accessibility.

## Workflow checklist

- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md)
